internal/pkg/mapper/sharedcode/helper/helper.go

```diff
@@ -128,61 +128,61 @@ func (h *SharedCodeHelper) GetSharedCodeKey(object model.Object) (*model.Config,
 	return transformation, sharedCodeKey, nil
 }
 
-func (h *SharedCodeHelper) GetSharedCodeByPath(branchKey model.BranchKey, path string) *model.ConfigState {
+func (h *SharedCodeHelper) GetSharedCodeByPath(branchKey model.BranchKey, path string) (*model.ConfigState, error) {
 	// Get branch
 	branch, found := h.state.Get(branchKey)
 	if !found {
-		return nil
+		return nil, fmt.Errorf(`%s not found`, branchKey.Desc())
```

Comment:
Pri debugovani som si vsimol, ze by bolo dobre vylepsit chyby ohladom shared codes.
Ono tam chybala hlavne plna cesta ku shared code + plna cesta, odkial sa to referencuje.